### PR TITLE
Fix add ring/release_type selection option

### PR DESCRIPTION
**Pull request by zhongfly**
Fix https://github.com/LSPosed/MagiskOnWSA/pull/283
Use "Retail" as rollback value

### CODE REVIEW

File: .github/workflows/magisk.yml [hunk 1]
```diff
@@ -110,9 +110,10 @@ jobs:
           import urllib.request
 
           arch = "${{ matrix.arch }}"
-          build_type_map = {"retail": "Retail", "release preview": "RP", "insider slow": "WIS"}
+          release_type_map = {"retail": "Retail", "release preview": "RP", "insider slow": "WIS"}
```
Comment (PeterNjeim) [inline .github/workflows/magisk.yml:113]: "insider fast": "WIF" could also be added with this PR

Comment (zhongfly) [reply .github/workflows/magisk.yml:113]: > "insider fast": "WIF" could also be added with this PR

I will add it

**Comment by zhongfly** [comment]
The release type in the artifact name seems inaccurate?

**Comment by PeterNjeim** [comment]
What do you mean? `WSA-with-magisk-NoGApps_2203.40000.1.0_x64_Release-Nightly`. Are you referring to the `Release-Nightly` part?

**Comment by zhongfly** [comment]
> What do you mean? `WSA-with-magisk-NoGApps_2203.40000.1.0_x64_Release-Nightly`. Are you referring to the `Release-Nightly` part?

Yes
Whether I choose retail or insider fast, the name of the artifact is still release-nightly.
It is also possible that these channels happen to be the same version now.

**Comment by PeterNjeim** [comment]
```yaml
a = html.find("a", string=re.compile("MicrosoftCorporationII\.WindowsSubsystemForAndroid_.*\.msixbundle"))
link = a["href"]
print(f"downloading link: {link}", flush=True)
out_file = "wsa.zip"
if not os.path.isfile(out_file):
    urllib.request.urlretrieve(link, out_file)

zip_name = ""
with zipfile.ZipFile(out_file) as zip:
    for f in zip.filelist:
        if arch in f.filename.lower():
            zip_name = f.filename
            if not os.path.isfile(zip_name):
                print(f"unzipping to {zip_name}", flush=True)
                zip.extract(f)
                ver_no = zip_name.split("_")
                ver = ver_no[1]
                with open(os.environ['GITHUB_ENV'], 'a') as g:
                    g.write(f'WSA_VER={ver}\n')
                rel = ver_no[3].split(".")
                rell = str(rel[0])
                with open(os.environ['GITHUB_ENV'], 'a') as g:
                    g.write(f'WSA_REL={rell}\n')
```

Relevant code. `WSA_REL={rell}` is what causes the `Release-Nightly` name

**Pull request merged**